DESCRIPTION

```diff
@@ -52,7 +52,8 @@ Imports:
     numberize,
     readr,
     rlang,
-    tibble
+    tibble,
+    utils
```

Comment:
No need to specify {utils} as a dependency as it is part of base R. You can double check by running running R CMD CHECK without after removing it.